source/menus/choose_file.py

```diff
@@ -63,17 +63,25 @@ def kinput(self, term: Interface, key: Union[int, None]) -> None:
 
         self.selected = max(0, min(self.selected, len(self.dirs) + len(self.files) - 1))
 
-    def click(self, term: Interface) -> State:
+    def click(self, term: Interface) -> Menu:
         """Handle a enter press, updating currently viewed files or changing state."""
         if self.selected < len(self.dirs):  # If it is a folder that is selected (they always come first)
             self.init_files(os.path.abspath(os.path.join(self.current_dir, self.dirs[self.selected])))
-            return State.file_explorer  # Don't change state
-
-        term.menus[State.playing] = Game(
-            os.path.abspath(
-                # The subtraction is to accomodate for directories being selected
-                os.path.join(self.current_dir, self.files[self.selected - len(self.dirs) - 1])
+            return self  # Don't change state
+
+        try:
+            return Game(
+                os.path.abspath(
+                    # The subtraction is to accomodate for directories being selected
+                    os.path.join(self.current_dir, self.files[self.selected - len(self.dirs) - 1])
```

Comment:
By the way, this caused an issue in selecting files that I've fixed in another branch. It should be:
```suggestion
                    os.path.join(self.current_dir, self.files[self.selected - len(self.dirs)])
```